pkgs/tools/misc/coreboot-utils/default.nix

```diff
@@ -0,0 +1,103 @@
+{ stdenv, fetchurl, zlib, pciutils, coreutils, acpica-tools, iasl, makeWrapper, gnugrep, gnused, file, buildEnv }:
+
+let
+  version = "4.10";
+
+  meta = with stdenv.lib; {
+    description = "Various coreboot-related tools";
+    homepage = "https://www.coreboot.org";
+    license = licenses.gpl2;
+    maintainers = [ maintainers.petabyteboy ];
+    platforms = platforms.linux;
+  };
+
+  generic = { pname, path ? "util/${pname}", ... }@args: stdenv.mkDerivation (rec {
+    inherit pname version meta;
+
+    src = fetchurl {
+      url = "https://coreboot.org/releases/coreboot-${version}.tar.xz";
+      sha256 = "1jsiz17afi2lqg1jv6lsl8s05w7vr7iwgg86y2qp369hcz6kcwfa";
+    };
+
+    enableParallelBuilding = true;
+
+    prePatch = ''
+      cd ${path}
+    '';
+
+    configureFlags = [
+      "INSTALL=install"
+      "PREFIX=${placeholder "out"}"
+    ];
+    installFlags = configureFlags;
```

Comment:
I don't think this is needed, since anything in `makeFlags` will already be there.

Reply:
Yup you're right!